### ISSUE TITLE
Docker isn't displaying Minecraft window

### ISSUE THREAD

**Issue by ryshng** [MineDojo/MineDojo#58]
I have installed MineDojo using the Docker image using wsl2 as a base engine. When executing vglrun python3 sample_script.py or python3, the output it's fine, but isn't showing anything.

**Comment by ryshng** [comment]
No, at the end I just installed it locally by cloning the whole code, but since I am not familiar with pip, I think the "problem" is there. If you are gonna try to install the minedojo locally using the 'pip install' command, it won't show you any window either. You need to clone the repo, then your sample can be run inside the project folder.

**Comment by dealexce** [comment]
I tried to use the docker image too because I met issues when I try to install locally, and I figured out how to display the Minecraft in docker in the end.
I think the reason it is not displaying is that by default the GUI in containers is not displaying to the host machine. So I referred [this post](https://gist.github.com/Mluckydwyer/8df7782b1a6a040e5d01305222149f3c).

First I installed [VcXsrv](https://sourceforge.net/projects/vcxsrv/) on Windows, and run it with the same configuration as step 2 in that post. This creates a X server instance that is able to receive and display the GUIs from clients.

Then I went into the container and run
```shell
export DISPLAY=host.docker.internal:0
export LIBGL_ALWAYS_INDIRECT=0
``` 
This 'redirects' (I'm not sure if it is precise, it's just my understanding) the GUI displays in your container to your host machine, so that the X server on your Windows can now receive the displays. Note that this command need to be run for every terminal, or you can append them into `~/.bashrc` and remember to run `source ~/.bashrc` for currently opening terminals.

After that, you can just `python3 sample_script.py` (no `vglrun` prepended) and you should be able to see the Minecraft window pops up.
